.github/workflows/ci.yaml

```diff
@@ -107,6 +107,11 @@ jobs:
           
       - name: Build dependencies 
         run: stack --install-ghc test --only-dependencies
+
+      - name: Set up Node
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
```

Comment:
@shayneczyzewski and @Martinsos The e2e tests are currently passing, but I assume GitHub will automatically bump its default version of Node to 18 when it comes out.

That's why it's probably a good idea to specify which Node version the CI should use. The change I made is based on [this](https://github.com/actions/setup-node#usage), but GitHub seems to be ignoring the file (i.e., it doesn't run the workflow). Any idea why?

Reply:
Hey @sodic good thinking pinning it in CI! I believe in this case it is just a bad UI for GitHub actions. It actually did attempt to run CI, but is failing: https://github.com/wasp-lang/wasp/actions/runs/1968440172

<img width="1415" alt="Screen Shot 2022-03-11 at 7 58 19 AM" src="https://user-images.githubusercontent.com/523636/157871008-cce40722-3527-4c2d-af03-321ff924b6a2.png">

```
[Error: .github#L1]
every step must define a `uses` or `run` key
```

I can try to take a look after morning meetings if still stuck 👍🏻

Reply:
I had a feeling something like this might be happening, thanks!

> I can try to take a look after morning meetings if still stuck 👍🏻

No worries, I'll take care of it. I was just missing diagnostics, and you showed me where to find them :)

Reply:
Awesome! 🚀

Reply:
They are running now @Martinsos it was a minor CI file syntax error